pages/project/[project_slug]/index.js

```diff
@@ -324,9 +320,31 @@ function GraphTooltip({ repository, tooltipData }) {
 }
 
 function Metrics({ id, repository }) {
+  const router = useRouter()
+  const { metric: queryKey } = router.query
```

Comment:
Instead of using a query key I wonder if we should make it an actual param `project/:project/:metric` instead?